evalml/pipelines/time_series_pipeline_base.py

```diff
@@ -221,8 +247,11 @@ def predict(self, X, objective=None, X_train=None, y_train=None):
             raise ValueError(
                 "Cannot call predict() on a component graph because the final component is not an Estimator."
             )
+        X = infer_feature_types(X)
+        self._validate_holdout_datasets(X, X_train)
         y_holdout = self._create_empty_series(y_train)
-        X, y_holdout = self._convert_to_woodwork(X, y_holdout)
+        y_holdout = infer_feature_types(y_holdout)
+        self._validate_holdout_datasets(X, X_train)
```

Comment:
Did you intend to validate the holdout datasets twice?

Reply:
My mistake lol